src/core.js

```diff
@@ -231,7 +233,36 @@ Core.prototype.init = function() {
     this._sceneFirstPass.add(this._meshFirstPass);
     this._sceneSecondPass.add(this._meshSecondPass);
     //this._sceneSecondPass.add(this._axes);
-    
+      
+////FramesPerSecond
+    /**
+     * @author mrdoob / http://mrdoob.com/
+     */
+//FPS-Start
+        var stats = new Stats();
+        stats.setMode( 0 ); // 0: fps, 1: ms, 2: mb
+
+        // align top-left
+        stats.domElement.style.position = 'absolute';
+        stats.domElement.style.left = '0px';
+        stats.domElement.style.top = '0px';
+
+        document.body.appendChild( stats.domElement );
+
+        var update = function () {
+
```

Comment:
avoid using excessive new lines. This leads to your whole code always using new lines. Something like, trying to highlight everything is equivalent to not highlighting anything at all.